spec/features/creating_peeps_spec.rb

```diff
@@ -0,0 +1,10 @@
+feature 'Adding a new message(peep)' do
+  scenario 'A user can post a message(peep) to chitter' do
+    visit('/messages/new')
+    fill_in('message', with: 'a new post')
+    fill_in('date-posted', with: '2022-01-12T19:30')
```

Comment:
Looks like this date-posted doesn't get stored in the database – is that something you were in the process of implementing?